src/server.py

```diff
@@ -437,25 +405,13 @@ def run_seta_query():
 @app.route("/data/setadetails/")
 @json_response
 def run_seta_details_query():
-    # TODO: remove inactive when buildbot api queries s/inactive/priority/
-    inactive = sanitize_bool(request.args.get("inactive", 0))
```

Comment:
I would like to leave a comment in here as this would break buildbot if we merged this or switched buildbot.  I am fine removing it though :)